lib/storage/src/data-chunk/readable-helper.ts

```diff
@@ -4,27 +4,25 @@ import { Readable } from "stream";
 import { DEFAULT } from "../upload/defaults";
 import { DataPart } from "./yield-chunk";
 
-interface StreamChunk {
-  Body: Buffer;
-  ended: boolean;
-}
-
 export async function* chunkFromReadable(reader: Readable, chunkSize: number): AsyncGenerator<DataPart, void, unknown> {
   let partNumber = DEFAULT.MIN_PART_NUMBER;
   let oldBuffer = Buffer.from("");
   while (partNumber < DEFAULT.MAX_PART_NUMBER) {
-    reader.resume();
-    const result = await _chunkFromStream(reader, chunkSize, oldBuffer);
-    reader.pause();
-
+    let currentBuffer = oldBuffer;
+    if(reader.readable) {
```

Comment:
Looks like prettier was not run on this code.
Either VSCode was not used, or prettier extension was not installed/disabled. And commit was created with `--no-verify` which didn't run pre-commit hooks to verify.

Do run prettier on this file for next revision, and create an issue if prettier failed to run because of some other reason.
 
```suggestion
    if (reader.readable) {
```

Reply:
yeah looks like Prettier isnt running for me. Strange. Created an issue https://github.com/aws/aws-sdk-js-v3/issues/1888

Reply:
reopening as it's not resolved.